draft-ietf-tls-dtls13.md

```diff
@@ -300,6 +307,44 @@ length:
 encrypted_record:
 : Identical to the encrypted_record field in a TLS 1.3 record.
 
+{:br}
+
+The short DTLS header format is:
+
+~~~~
+    struct {
+      uint16 short_epoch_and_sequence;  // 001ESSSS SSSSSSSS
+      opaque encrypted_record[remainder_of_datagram];
+    } DTLSShortCiphertext;
+~~~~
+
+The short_epoch_and_sequence document contains the epoch and sequence
+packed into a 16 bit integer as follows:
+
+- The first three bits are set to 001 in order to allow multiplexing
+  between DTLS and VoIP protocols (STUN, RTP/RTCP, etc.) {{?RFC7983}}
+  and distinguish the short from long header formats.
+
+- The fourth bit is the low order bit of the epoch value.
+
+- The remaining bits contain the low order 12 bits of the sequence
+  number.
+
+For deprotecion purposes, the nonce is computed using the full
+sequence number, which can be reconstructed as described in
+{{reconstructing}}.
+
+In this format, the length field is omitted and therefore the
+record consumes the entire rest of the datagram in the lower
+level transport. It is not possible to have multiple
+DTLSShortCiphertext format records in the same datagram.
+
+DTLSShortCiphertext MUST only be used for data which is protected with
+one of the application_traffic_secrets, and not for either
+handshake or early data. When using the application_traffic_secret,
```

Comment:
Maybe "an application_traffic_secret for message protection" to be consistent with the previous sentence?